test/test_forms.jl

```diff
@@ -48,4 +51,19 @@ using Random
 
   @test underlying_graph(ket_network(qf)) == underlying_graph(ψket)
   @test underlying_graph(operator_network(qf)) == underlying_graph(A)
+
+  ∂qf_∂v = only(derivative_state(qf, [v]))
+  @test (∂qf_∂v) * (qf[ket_vertex_map(qf)(v)] * qf[bra_vertex_map(qf)(v)]) ≈
```

Comment:
Maybe we should have a shorthand `ket_vertex(qf, v)` for spelling `ket_vertex_map(qf)(v)`.

Reply:
Sounds sensible!